NEW_MODULE_TECH_EVAL.MD

```diff
@@ -62,10 +71,13 @@ Information gathered includes:
     * Weigh in on any subjective criteria.
     * The Technical Council may ask the Evaluator(s) to make changes to the evaluation.  
     * This review may happen either out of band, during the regularly scheduled Technical Council meeting, or a separate meeting scheduled specifically for this purpose.
-1. The Technical Council decides, using its standard protocols, to accept or reject the module.
+1. The Technical Council decides, using its standard protocols, to accept, provisionally accept, or reject the module, or else to defer a decision pending other action.
     * An evaluation where all criteria are met will typically result in the acceptance of a module by the Technical Council.
-    * An evaluation where not all criteria are met will typically result in the rejection of a module by the Technical Council.
-    * When the Technical council's decision is contrary to the evaluation, the Technical Council is required to provide written justification for their decision.
+    * An evaluation where not all criteria are met will typically result in the rejection of a module by the Technical Council. However:
+        * If some of the failures are due to proposed architectural (or other cross-module) changes, the TC may request that Submitter first propose those changes via the RFC process to get sufficient community input. In that situation the TC may defer its decision pending the resolution of the RFC. (See [Before Development](#Before-Development).)
+        * If TC and Submitter unanimously agree on module changes that would resolve any failures, the TC may decide to provisionally accept the module with an agreed-upon timeline for the changes to be completed.  When Submitter notifies TC that the changes are complete, the reviewer(s) may update the evaluation and TC may adjust its decision.
```

Comment:
This provisional acceptance option for minor issues is solely to prevent rejecting a module (and potentially missing a release deadline) for something we unanimously agree is an easy fix.  It allows us to document that these (minor) issues *do* need to be addressed, for the overall TCR goals, rather than simply accepting the module "even though" these problems exist.

Reply:
> This provisional acceptance option for minor issues

Is provisional acceptance only intended for _minor_ issues?

If so, how does this fit with the process applied for the list related modules (which I assumed this formalisation had come from)?

Reply:
> Is provisional acceptance only intended for minor issues?

Yes -- my word "minor" being shorthand for what the text refers to as unanimous agreement among TC members and the submitter.  We may want to make that clearer.  

The intent is for obvious things everyone agrees on, which we do seem to have a few of on each review.  This prevents delay where there's not a single person who thinks it is needed.

The list-related modules issue about database constraints would not have met this threshold as it was far from unanimous.

Reply:
> When does provisionally acceptance become acceptance (or rejection)?

Hopefully the explanation above justifies that it should be a very quick process to remediate issues we all unanimously agree on.  I am guessing perhaps a week or two max.  What we say is just "When Submitter notifies TC that the changes are complete, the reviewer(s) may update the evaluation and TC may adjust its decision."